specs/templates/custom-component.md

```diff
@@ -1,15 +1,15 @@
-# Nimble Component Name [Template]
+# Component Name [Template]
 
 ## Overview
 
 *The name of the component, along with a high-level description.*
 
 ### Background
 
-*Relevant historical or background information*
-*Link to Interaction Design spec*
-*Link to Visual Design spec*
-*Link to relevant work items, related existing issues, etc.*
+- *Relevant historical or background information*
+- *Link to Interaction Design spec [Optional for Spright]*
+- *Link to Visual Design spec [Optional for Spright]*
+- *Link to relevant work items, related existing issues, etc.*
 
```

Comment:
We need a section that's about choosing which component library it will be a part of and for the justification to be provided.